packages/matrix-identity-server/src/db/sql/sql.ts

```diff
@@ -25,7 +25,10 @@ const tables: Record<Collections, string> = {
   roomTags:
     'id varchar(64) PRIMARY KEY, authorId varchar(64), content text, roomId varchar(64)',
   userHistory: 'address text PRIMARY KEY, active integer, timestamp integer',
-  userQuotas: 'user_id varchar(64) PRIMARY KEY, size int'
+  userQuotas: 'user_id varchar(64) PRIMARY KEY, size int',
+  // Key management
+  longTermKeypairs: 'keyID varchar(64) PRIMARY KEY, public text, private text',
+  shortTermKeypairs: 'keyID varchar(64) PRIMARY KEY, public text, private text'
```

Comment:
Such mono entries could go into `keys` table, couldn't it ?

Reply:
Then how to know which key is active ?

Reply:
For now, we haven't encountered a case where we had to deal with invalid keys so I considered a key to be valid when it is stored in the database (hence a deleteKey method). 
Is active and valid the same according to you ?

Reply:
No:
- valid includes "previous": all existing object signed with any "valid" keys can be verified
- active means currently in use